service-front/module/Application/src/Controller/CPFlowController.php

```diff
@@ -69,16 +69,31 @@ public function howWillCpConfirmAction(): ViewModel|Response
                 );
                 $view->setVariables($formProcessorResponseDto->getVariables());
             } else {
-                $this->opgApiService->updateIdMethod($uuid, $formData['id_method']);
-                if ($formData['id_method'] == 'po') {
+                if ($formData['id_method'] == \Application\Enums\IdMethod::PostOffice->value) {
```

Comment:
Nicer to import these classes with `use` rather than putting the FQCN in the codebase